_layouts/default.html

```diff
@@ -1,17 +1,380 @@
+<html lang="{{ page.lang | default: site.lang | default: "en" }}">
+
+  {%- include head.html -%}
+
+  <body>
+
+    {%- include header.html -%}
+    
+    <div class="flex" aria-label="Content">
+      <div class="container herounit">
+        <div class="flex flex--vertical content">
+          <h3 class="flex">Cylc (“silk”) is a workflow engine for cycling systems</h3>
+          <p>Cylc was originally developed for operational environmental forecasting at <a href="http://www.niwa.co.nz/">NIWA</a> by <a href="hilary.oliver@niwa.co.nz">Dr Hilary Oliver</a> - however it is not specialized to forecasting in any way. It is now an Open Source collaboration involving NIWA, <a href="http://www.metoffice.gov.uk/">Met Office</a>, and <a href="https://github.com/cylc/cylc-flow/blob/master/CONTRIBUTING.md#code-contributors">others</a>. It is available under the <a href="{{'/license'}}">GPL v3 license</a>.</p>
+          <div class="flex download-buttons">
+            <div class="flex flex--vertical m-r">
+              <label>Production</label>
+              <a class="flex grow" id="cylc7-button">
+                <button class="button flex grow">
+                    <img src={{'/assets/icon-download.svg'}} />
+                    <span id="cylc7-latest-release-name"></span>
+                </button>
+              </a>
+              <label id="cylc7-latest-release-pubdate-text"></label>
+            </div>
+            <div class="flex flex--vertical m-l">
+                <label>Preview</label>
+                <a class="flex grow" id="cylc8-button">
+                  <button class="button flex grow">
+                    <img src={{'/assets/icon-download.svg'}} />
+                    <span id="cylc8-latest-release-name"></span>
+                  </button>
+                </a>
+                <label id="cylc8-latest-release-pubdate-text"></label>
+              </div>
+          </div>
+        </div>
+        <div class="flex flex--vertical release jc-space-around">
+            <div class="flex flex--vertical">
+              <h3 class="flex jc-center">Next Release:&nbsp;
+                <a class="page-link" id="milestone-url" target="_blank"><span id="milestone-version"></span> (<span id="progress-info"></span>)</a>
+              </h3>
+              <div class="flex">
+                <div class="progressbar" id="milestone-container">
+                  <div class="progress" id="progress"></div>
+                  <div class="flex label jc-space-between">
+                    <span>0%</span>
+                    <span>100%</span>
+                  </div>
+                </div>
+              </div>
+            </div>
+            <div class="flex flex--vertical">
+              <div class="flex ai-center jc-space-between">
+                <div class="flex">
+                  <h4 class="flex"><b>Previous Release</b></h4>
+                </div>
+                <a href="https://github.com/cylc/cylc.github.io" target="_blank">
```

Comment:
This now points to the web site source repository, not the cylc-flow code repository.  (However, we will remove this button - see the current non-responsive site: https://cylc.github.io/)